app/models/game.rb

```diff
@@ -0,0 +1,8 @@
+class Game < ActiveRecord::Base
+  validates :week_id, presence: true
+  validates :home_team_id, presence: true
```

Comment:
Once the associations are in, I believe you can shorten these to `validates :home_team ...`, etc and leave off the id.